### PR TITLE
Add '@module' comment for JSDoc

### PR DESCRIPTION
**Pull request by cuobiezi**

### CODE REVIEW

**Comment by kt3k** [comment]
@cuobiezi Can you sign the CLA from the above link? Is @likenanhai your account? The user @likenanhai also needs to sign the CLA to land this PR

**Comment by cuobiezi** [comment]
> @cuobiezi Can you sign the CLA from the above link? Is @likenanhai your account? The user @likenanhai also needs to sign the CLA to land this PR

> @Kt3k My Account cuobiezi signed , But the I'm not sure If I can sign @likenanhai account again. I will try . 
> How should I do ? If I can't sing @likenanhai account  ? Thanks.

File: uuid/mod.ts [hunk 1]
```diff
@@ -3,6 +3,10 @@
 
 // Based on https://github.com/kelektiv/node-uuid -> https://www.ietf.org/rfc/rfc4122.txt
 // Supporting Support for RFC4122 version 1, 4, and 5 UUIDs
+/**
+ * UUid
```
[the remark this answers is not in the document]
Comment (cuobiezi) [reply uuid/mod.ts]: > @kt3k Done , Thanks for you reivew.

**Comment by kt3k** [comment]
> @kt3k My Account cuobiezi signed , But the I'm not sure If I can sign @likenanhai account again. I will try .
> How should I do ? If I can't sing @likenanhai account  ? Thanks.

Can you squash 3 commits into single commit with your current git settings? If that's difficult, maybe please create a new PR with the same change with your current git settings.

BTW the email address you use for git commit doesn't seem connected to your github account. Please add that email address to your github account. ref: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/adding-an-email-address-to-your-github-account

**Comment by cuobiezi** [comment]
@kt3k Moved to #2190

**Comment by kt3k** [comment]
> @kt3k Moved to https://github.com/denoland/deno_std/pull/2190

Thanks!

**Pull request closed without merging**